test/build/base-url/base-url.test.js

```diff
@@ -21,15 +26,19 @@ describe('buildOptions.baseUrl', () => {
 
   it('import.meta.env works', () => {
     // env is present in index.js
-    expect(indexJS).toEqual(
+    expect(files['/index.js']).toEqual(
```

Comment:
This is a totally-optional test helper for reading files. You pass in any array of files to `readFiles([...files], { cwd })`, and you get back a map of `{ [fileName]: "contents" }`. I found it helpful in reading a bunch of files at once, but without the annoyance of snapshots